### PR TITLE
Sqlite

### PR DESCRIPTION
**Pull request by mezgoodle**
Useful links:

- [Help Wiki](https://github.com/mezidia/grechka.com/wiki/MongoDB)
- [Mongoose documentation](https://mongoosejs.com/docs/guide.html)

### CODE REVIEW

**Comment by gazinaft** [comment]
@mezgoodle what's the reason of DB export update?

**Comment by mezgoodle** [comment]
> @mezgoodle what's the reason of DB export update?

I don't understand you

**Comment by gazinaft** [comment]
> > @mezgoodle what's the reason of DB export update?
> 
> I don't understand you

I mean why did you changed to usual export from export default

**Comment by mezgoodle** [comment]
Help Wiki: https://github.com/mezidia/grechka.com/wiki/MongoDB

**Comment by mezgoodle** [comment]
In test.js I show how to work with MongoDB, and lower there are examples. Also I created wiki-page with API

```
![Снимок](https://user-images.githubusercontent.com/41520940/105469830-75285680-5ca1-11eb-815e-dab1522886bd.PNG)

Add category: 
{ _id: 600a9380a8889f3ecc68724e, categoryType: 'ketchups', __v: 0 }

Add manufacturer: 
{
  _id: 600a94118f50ad381435aa4e,
  companyName: 'ATB',
  country: 'Ukraine',
  __v: 0
}

Add package: 
{ _id: 600a94473f58732eb0118eff, packageType: 'Box', __v: 0 }

Founded category with id: 600a9380a8889f3ecc68724e
Founded manufacturer with id: 600a94118f50ad381435aa4e
Founded package with id: 600a94473f58732eb0118eff
Add product: 
{
  _id: 600a9462d7608e0720d4959e,
  productName: 'Car',
  categoryId: 600a9380a8889f3ecc68724e,
  weight: 3200,
  packageId: 600a94473f58732eb0118eff,
  priceSegment: 'Low',
  productURL: 'https://google.com.ua',
  productImgURL: 'https://google.com.ua/images',
  description: 'Renault Logan black color',
  __v: 0
}

Founded product with id: 600a9462d7608e0720d4959e
Add cost: 
{
  _id: 600a94881cdf6a33b4c0d39a,
  productId: 600a9462d7608e0720d4959e,
  price: 9000000,
  date: 2021-01-22T09:02:00.372Z,
  __v: 0
}

Add history: 
{
  _id: 600a94881cdf6a33b4c0d39b,
  productId: 600a9462d7608e0720d4959e,
  price: 9000000,
  date: 2021-01-22T09:02:00.377Z,
  __v: 0
}

Product deleted: 
{
  n: 1,
  opTime: {
    ts: Timestamp { _bsontype: 'Timestamp', low_: 5001, high_: 1611311064 },
    t: 2
  },
  electionId: 7fffffff0000000000000002,
  ok: 1,
  '$clusterTime': {
    clusterTime: Timestamp { _bsontype: 'Timestamp', low_: 5001, high_: 1611311064 },
    signature: { hash: [Binary], keyId: [Long] }
  },
  operationTime: Timestamp { _bsontype: 'Timestamp', low_: 5001, high_: 1611311064 },
  deletedCount: 1
}

Product updated
{
  n: 1,
  nModified: 1,
  opTime: {
    ts: Timestamp { _bsontype: 'Timestamp', low_: 1, high_: 1611311558 },
    t: 2
  },
  electionId: 7fffffff0000000000000002,
  ok: 1,
  '$clusterTime': {
    clusterTime: Timestamp { _bsontype: 'Timestamp', low_: 1, high_: 1611311558 },
    signature: { hash: [Binary], keyId: [Long] }
  },
  operationTime: Timestamp { _bsontype: 'Timestamp', low_: 1, high_: 1611311558 }
}
```

**Comment by mezgoodle** [comment]
Снимок

File: scripts/node-js/db.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const sqlite3 = require('sqlite3').verbose();
+
+class DB{
+  // Open database in memory
+  constructor (path) {
+    this.db = new sqlite3.Database(path, sqlite3.OPEN_READWRITE, (err) => {
+      if (err) {
+        return console.error(err.message);
```
[the remark this answers is not in the document]
Comment (mezgoodle) [reply scripts/node-js/db.js]: Outdated

**Comment by mezgoodle** [comment]
[test.js](https://gist.github.com/mezgoodle/2c3bd95e4bf0814e4a2917dcf72a3e82)

**Pull request merged**